plugin/storage/badger/options.go

```diff
@@ -19,18 +20,33 @@ type Options struct {
 	// This storage plugin does not support additional namespaces
 }
 
-// NamespaceConfig is badger's internal configuration data
+// NamespaceConfig is badger's internal configuration data.
```

Comment:
are there any groupings we want to make in this configuration?

Reply:
I can think of a couple
  * directories, as per comment above
  * TTL. Now it could have a single sub-element for span_store, but later we can add values for dependencies, sampling, etc.